app/uk/gov/hmrc/gform/LookupLoader.scala

```diff
@@ -17,29 +17,39 @@
 package uk.gov.hmrc.gform
 
 import com.miguelfonseca.completely.AutocompleteEngine
-import com.miguelfonseca.completely.text.analyze.tokenize.WordTokenizer
 import com.miguelfonseca.completely.text.analyze.transform.LowerCaseTransformer
 import kantan.csv._
 import kantan.csv.ops._
 import uk.gov.hmrc.gform.sharedmodel.LangADT
 import uk.gov.hmrc.gform.sharedmodel.formtemplate.Register
-import uk.gov.hmrc.gform.lookup.{ AjaxLookup, LocalisedLookupOptions, LookupAdapter, LookupId, LookupInfo, LookupLabel, LookupOptions, LookupRecord, LookupType, RadioLookup, ShowAll }
+import uk.gov.hmrc.gform.lookup._
 
 class LookupLoader {
 
-  implicit private val lookupIdCellDecoder: CellDecoder[LookupId] = implicitly[CellDecoder[String]].map(LookupId)
+  implicit private val lookupIdCellDecoder: CellDecoder[LookupId] =
+    implicitly[CellDecoder[String]].map(s => LookupId(s.trim))
   implicit private val lookupLabelCellDecoder: CellDecoder[LookupLabel] =
     implicitly[CellDecoder[String]].map(s => LookupLabel(s.trim))
+  implicit private val lookupPriorityCellDecoder: CellDecoder[LookupPriority] =
+    implicitly[CellDecoder[Option[Int]]].map(p => LookupPriority(p.getOrElse(1)))
+  implicit private val lookupRegionCellDecoder: CellDecoder[LookupRegion] =
+    implicitly[CellDecoder[Int]].map(r => LookupRegion(r))
+  implicit private val lookupKeyWordsCellDecoder: CellDecoder[LookupKeyWords] =
+    implicitly[CellDecoder[Option[String]]].map(s => LookupKeyWords(s.map(_.trim)))
```

Comment:
I dont think there is need to use Option. Lookupkeywords(keywords: String) is just fine and LookupKeywords("") could represent default. Could define a constant just like LookupPriority